From 021c5060d5151d0882fc795327a6e61e07758d05 Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Fri, 4 Jan 2019 13:56:12 +0200
Subject: [PATCH 0976/1200] mvebu: sar: ap807: update clocking information

For the ap807 based system cpu_freq==6 is reserved for
CPU_2200_DDR_1200_RCLK_1200 mode. Reflect it in the code.

Change-Id: I23e9ada93e8201f43ef58d656f9f10f9e6f3c32e
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/62426
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/1880
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Kostya Porotchkin <Kostya.Porotchkin@cavium.com>
---
 drivers/misc/mvebu_sar/ap807_sar.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/misc/mvebu_sar/ap807_sar.c b/drivers/misc/mvebu_sar/ap807_sar.c
index 7a13335118..4387befa3e 100644
--- a/drivers/misc/mvebu_sar/ap807_sar.c
+++ b/drivers/misc/mvebu_sar/ap807_sar.c
@@ -109,7 +109,7 @@ struct sar_info ap807_sar_1[] = {
 enum clocking_options {
 	CPU_2000_DDR_1200_RCLK_1200 = 0x0,
 	CPU_2000_DDR_1050_RCLK_1050 = 0x1,
-	CPU_1800_DDR_1200_RCLK_1200 = 0x6,
+	CPU_2200_DDR_1200_RCLK_1200 = 0x6,
 	CPU_1800_DDR_1050_RCLK_1200 = 0x7,
 	CPU_1600_DDR_1200_RCLK_1200 = 0x0d,
 	CPU_1600_DDR_900_RCLK_900 = 0x0e,
@@ -120,7 +120,7 @@ static const u32 pll_freq_tbl[16][4] = {
 	/* CPU */   /* DDR */   /* Ring */
 	{2.0 * GHz, 1.2  * GHz, 1.2  * GHz, CPU_2000_DDR_1200_RCLK_1200},
 	{2.0 * GHz, 1.05 * GHz, 1.05 * GHz, CPU_2000_DDR_1050_RCLK_1050},
-	{1.8 * GHz, 1.2  * GHz, 1.2  * GHz, CPU_1800_DDR_1200_RCLK_1200},
+	{2.2 * GHz, 1.2  * GHz, 1.2  * GHz, CPU_2200_DDR_1200_RCLK_1200},
 	{1.8 * GHz, 1.05 * GHz, 1.2 * GHz, CPU_1800_DDR_1050_RCLK_1200},
 	{1.6 * GHz, 1.2 * GHz, 1.2 * GHz, CPU_1600_DDR_1200_RCLK_1200},
 	{1.6 * GHz, 0.9  * GHz, 0.9  * GHz, CPU_1600_DDR_900_RCLK_900},
-- 
2.22.0

